### PR TITLE
Adds styles for author modal

### PR DESCRIPTION
**Pull request by jon-stevens**
Adds some styling for the author list. 
Adds the image for the pointy thing on the bottom of the box.

Screenshot 2021-09-29 at 11 16 32

Screenshot 2021-09-29 at 11 16 37

### CODE REVIEW

File: toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss [hunk 1]
```diff
@@ -52,6 +52,22 @@ $author-icon-background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org
 
 		.js & {
 			display: inline-block;
+
+			&.corresponding {
+				margin-right: 20px;
```
Comment (benjclark) [inline toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss:57]: Is there a plan to start using the spacing() function in this brand toolkit?

File: toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss [hunk 2]
```diff
@@ -52,6 +52,22 @@ $author-icon-background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org
...
+
+				&:after {
+					content: ' ';
+					display: inline-block;
+					position: absolute;
+					background: $author-icon-background;
```
Comment (benjclark) [inline toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss:63]: I realise it's not part of this PR but I noticed that this variable is defined in the 50-components file when it should be in the 10-settings file for this

File: toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss [hunk 3]
```diff
@@ -132,6 +134,20 @@ $author-icon-background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org
 			box-shadow: 1px 1px 5px rgba(0, 0, 0, .2);
 			padding: $submission-header--padding;
 			margin: -15px 30px 30px 0;
+
+			&:after {
+				content: ' ';
+				width: 30px;
```
[the remark this answers is not in the document]
Comment (jon-stevens) [reply toolkits/springernature/packages/springernature-submission-header/scss/50-components/enhanced.scss]: Yep, will do

**Review by benjclark — APPROVED** [review]
